filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
```

Comment:
Could you please point out where in Ansible this is documented behavior? I've looked at `filter_plugins` in the source tree and haven't seen anything that requires appending `a` to `*args` or similar.